packages/hash/api/src/model/util.ts

```diff
@@ -266,15 +268,20 @@ export const propertyTypeInitializer = (
   };
 };
 
-export type EntityCreatorParams = {
+export type EntityTypeCreatorParams = {
   namespace: string;
   title: string;
   properties: {
-    baseUri: string;
+    baseUri: BaseUri;
     versionedUri: string;
     required?: boolean;
     array?: { minItems?: number; maxItems?: number } | boolean;
   }[];
+  outgoingLinks: {
+    versionedUri: string;
```

Comment:
Maybe it's worth us changing these to be more explicit. Such as `linkTypeId` (or versioned URI if you'd prefer) and above we could have `propertyTypeId` (assuming we combine the `baseUri` and `versionedUri` like I suggested in another comment)

Reply:
> or versioned URI if you'd prefer

Isn't `versionedUri` what it already is? Or am I looking at the wrong field. Happy to change it to `linkTypeId` and `propertyTypeId` if you'd prefer, just not sure what you're asking for.

Reply:
Sorry I mean `linkTypeId` or `linkTypeVersionedUri` if you prefer that

Reply:
I've renamed the arguments in https://github.com/hashintel/hash/pull/1068/commits/5b28cbef8a39844a7c802369708bc7cf2a523041